### ISSUE TITLE
How can I add ellipsis to text if it is too long?

### ISSUE THREAD

**Issue by Drieger** [voilab/voilab-pdf-table#28]

**Comment by tafel** [comment]
Just to see if it does what you want, could you try to patch your file `voilab-table.js` at line `187` in your node modules :

Add `|| column.ellipsis`  like below.

```
...
186   content = renderer ? renderer(self, row, false, column) : row[column.id],
187   height = !content || column.ellipsis ? 1 : self.pdf.heightOfString(content, lodash.assign(lodash.clone(column), {
188       width: column.width - getPaddingValue('horizontal', column.padding)
...
```
Should this to the trick?

**Comment by Drieger** [comment]
@tafel, thats exactly it. Should I create a PR with the proposed change?

**Comment by tafel** [comment]
woops, I just did the release too much quicker than I should :-P Forgot to update the changelog.

But anyway, I released a 0.4.2 version with this option.

**Comment by Drieger** [comment]
Great, thanks.
